src/sensor.cpp

```diff
@@ -265,6 +265,22 @@ namespace librealsense
         }
     }
 
+    stream_profiles sensor_base::get_stream_profiles(int tag) const
+    {
+        if (tag == profile_tag::PROFILE_TAG_ANY)
+            return *_profiles;
+
+        stream_profiles profiles;
```

Comment:
Consider renaming to something like `result`. `profiles` is confusing with `_profiles`

Reply:
done